# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: architecture/powerpc64/pkg_binutils.diff
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- binutils-2.31.1/ld/configure.tgt.vanilla	2018-12-30 15:39:51.612864737 +0000
+++ binutils-2.31.1/ld/configure.tgt	2018-12-30 15:45:27.336859688 +0000
@@ -549,9 +549,9 @@
   | powerpc*-*-solaris* | powerpc*-*-kaos* | powerpc*-*-vxworks*)
 			case "${targ}" in
 			powerpc64*)
-			    targ_emul=elf64ppc
-			    targ_extra_emuls="elf32ppc elf32ppclinux elf32ppcsim"
-			    targ_extra_libpath="elf32ppc elf32ppclinux" ;;
+			    targ_emul=elf32ppclinux
+			    targ_extra_emuls="elf64ppc elf32ppc elf32ppcsim"
+			    targ_extra_libpath="elf64ppc elf32ppc" ;;
 			*linux*)
 			    targ_emul=elf32ppclinux
 			    targ_extra_emuls="elf32ppc elf32ppcsim"
